lifecycle.md

```diff
@@ -46,13 +46,14 @@ For a mature standard, the whole process can repeat for proposed changes. New id
 
 As the standard, test suite and implementations co-evolve, an increasing number of tests will pass everywhere. For those tests, however trivial, full interop has been achieved. Once achieved, tooling should make it impossible to accidentally depart from full interop. Web developers can depend on the feature and delight their users, in any browser.
 
-## What about?
+## Examples
+ * DOM: mature [standard](https://dom.spec.whatwg.org/) and [test suite](https://github.com/w3c/web-platform-tests/tree/master/dom).
+ * HTML: the messagerror event had [spec text](https://github.com/whatwg/html/pull/2530) and [tests](https://github.com/w3c/web-platform-tests/pull/5567) written before any implementation.
+ * Trusted Types: [explainer](https://github.com/mikewest/trusted-types/blob/master/README.md) and [tentative tests](https://github.com/w3c/web-platform-tests/tree/cbc2da38b90e0870ac50a205d2fc2773de41bd5d/trusted-types).
 
-### Writing tests even earlier?
+*Note: Whether writing tests first or implementing first is most productive depends on circumstances. The most important thing is that tests are written and shared, not the order.*
```

Comment:
I think we do have a quality problem of sorts still. It's often the case that when there's an initial implementation and specification, and maybe some tests, there's still many things found by the second implementation. Not entirely sure how to address that though, but we need some kind of incentive for folks to perform their due diligence.

Reply:
Yes, the temptation to assume tests are good and not think too critically about them is pretty strong. Not sure what to do about it, apart from sketching what I think good behavior is under "Second implementation"